Mods/interface/PuppiMod.h

```diff
@@ -57,17 +53,26 @@ namespace mithep
       void SetDump( Bool_t dump )                            { fDumpingPuppi = dump;           }
 
     protected:
-      void                  SlaveBegin();
-      void                  SlaveTerminate();
-      void                  Process();
+      void                  SlaveBegin() override;
+      void                  SlaveTerminate() override;
+      void                  Process() override;
+
+      enum ParticleType {
```

Comment:
Direct numeric values were used for particle types, and that is simply put a bad coding style (relic of C/Fortran era). Enum is there to exactly solve these problems. See the implementation of GetParticleType().